src/python/foglamp/device/__main__.py

```diff
@@ -5,36 +5,88 @@
 # See: http://foglamp.readthedocs.io/
 # FOGLAMP_END
 
+""" Starts the device server """
+
 import sys
+import argparse
 
+import foglamp.device.exceptions as exceptions
+from foglamp import logger
 from foglamp.device.server import Server
 
-"""Starts the device server"""
-
 __author__ = "Terris Linenbach"
 __copyright_ = "Copyright (c) 2017 OSIsoft, LLC"
 __license__ = "Apache 2.0"
 __version__ = "${VERSION}"
 
+_MODULE_NAME = "device_service"
+
+_MESSAGES_LIST = {
+
+    # Information messages
+    "i000000": "",
+
+    # Warning / Error messages
+    "e000000": "generic error.",
+    "e000001": "cannot proceed the execution, invalid device service name - service name |{0}|  error details |{1}|",
+    "e000002": "cannot proceed the execution, missed device service name",
+    "e000003": "cannot complete the operation - error details |{0}|",
+}
+""" Messages used for Information, Warning and Error notice """
```

Comment:
@sstefano69 It will be great to have exceptions.py as src/python/foglamp/storage/exceptions.py with code and message and log while catching / raising these exceptions; eventually better will be to allow each package has a util(s).py and exception(s).py including root python package